securing_apps/topics/oidc/java/fuse7/cxf-builtin.adoc

```diff
@@ -1,10 +1,10 @@
 
 [[_fuse7_adapter_cxf_builtin]]
-===== Securing an Apache CXF Endpoint on the Default Jetty Engine
+===== Securing an Apache CXF Endpoint on the Default Undertow Engine
 
-Some services automatically come with deployed servlets on startup. One such service is the CXF servlet running in the $$http://localhost:8181/cxf$$ context. Securing such endpoints can be complicated. One approach, which {project_name} is currently using, is `ServletReregistrationService` which undeploys a built-in servlet at startup, enabling you to redeploy it on a context secured by {project_name}.
+Some services automatically come with deployed servlets on startup. One such service is the CXF servlet running in the $$http://localhost:8181/cxf$$ context. Fuse's Pax Web supports context configuration via Configuration Admin which can be used to secure endpoints by {project_name}.
```

Comment:
_context configuration_ is not precise - it's rather _altering_/_processing_ of existing contexts. You can refer to https://ops4j1.jira.com/wiki/spaces/paxweb/pages/354025473/HTTP+Context+processing